app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    if isinstance(command, str) and command.split(" ")[0] == "cp":
+        file_name = command.split(" ")[1]
+        destination_file = command.split(" ")[2]
```

Comment:
The `command.split(" ")` method is called multiple times. To improve efficiency, split the command once and store the result in a variable. Then, use the elements of this list to access `file_name` and `destination_file`.